src/js/tracker.js

```diff
@@ -938,6 +962,31 @@
 		}
 
 		/**
+		 * Get data for OptimizelyX contexts - active experiments on current page
+		 * 
+		 * @returns Array content of lite optimizely lite context
+		 */
+		function getOptimizelyXSummary() {
+			var state = getOptimizelyXData('state');
+			var experiment_ids = state.getActiveExperimentIds();
+			var experiments = getOptimizelyXData('data', 'experiments');
+			var visitor = getOptimizelyXData('visitor');
+
+			return lodash.map(experiment_ids, function(activeExperiment) {
+				variation = state.getVariationMap()[activeExperiment];
+				variationName = variation.name;
+				variationId = variation.id;
+				visitorId = visitor.visitorId;
+				return {
+					experimentId: parseInt(activeExperiment),
+					variationName: variationName,
+					variation: parseInt(variationId),
```

Comment:
I've only named it `variation` to be consistent with the existing `optimizely_summary` schema, happy to change it to `variationId` though if it makes more sense.